### DISCUSSION TITLE
Welcome to Hyper Discussions!

### DISCUSSION THREAD

**Discussion by leerob** [vercel/hyper#5646] [Announcements]
Hey everyone! 👋 

Thank you to everyone who continues to use, support, and love Hyper. This project wouldn't exist without you! We know you've been asking for new releases (including things like support for M1 Macs, new themes, security fixes, and more) and we have been listening. We recently published a [new canary release](https://github.com/vercel/hyper/releases/tag/v3.1.0-canary.5) and will continue to publish releases on a more regular cadence now. 

We're also turning on GitHub Discussions to allow y'all to have a place to ask questions and get answers! Please consider opening a discussion if you have a question versus an issue. Issues will be a better place for bugs with some way to reproduce or investigate. 

Thank you ❤️

**Comment by ivyreese** [comment]
Thank you for keeping the dream alive. I've been running the canary build on my M1 Air and it's _fantastic!_ No issues, blazing performance. Kudos.

**Comment by wren** [comment]
@leerob Thank you for doing all this work. I just rediscovered Hyper and am very excited to have it back in my toolkit.

That being said, I know how much work it can be to take over an open source project like this. So, if someone wanted to help out, where would you say you need the most help?
